src/components/sdk/ContentShare/index.tsx

```diff
@@ -10,37 +10,49 @@ import { BaseSdkProps } from '../Base';
 
 interface Props extends BaseSdkProps {
   nameplate?: string;
+  tileId?: number;
 }
 
 export const ContentShare: React.FC<React.PropsWithChildren<Props>> = ({
   className,
+  tileId,
   ...rest
 }) => {
   const audioVideo = useAudioVideo();
-  const { tileId } = useContentShareState();
+  const contentShareState = useContentShareState();
+
+  // Use the provided tileId or fall back to the default (for backward compatibility)
+  const tileIdToRender =
+    tileId !== undefined ? tileId : contentShareState.tileId;
+
+  const attendeeId = tileIdToRender
+    ? contentShareState.tileIdToAttendeeId[tileIdToRender.toString()]
+    : null;
+
   const videoEl = useRef<HTMLVideoElement | null>(null);
 
   useEffect(() => {
-    if (!audioVideo || !videoEl.current || !tileId) {
+    if (!audioVideo || !videoEl.current || !tileIdToRender) {
       return;
     }
 
-    audioVideo.bindVideoElement(tileId, videoEl.current);
+    audioVideo.bindVideoElement(tileIdToRender, videoEl.current);
 
     return () => {
-      const tile = audioVideo.getVideoTile(tileId);
+      const tile = audioVideo.getVideoTile(tileIdToRender);
       if (tile) {
-        audioVideo.unbindVideoElement(tileId);
+        audioVideo.unbindVideoElement(tileIdToRender);
       }
     };
-  }, [audioVideo, tileId]);
+  }, [audioVideo, tileIdToRender]);
 
-  return tileId ? (
+  return tileIdToRender ? (
     <ContentTile
       objectFit="contain"
-      className={className || ''}
+      className={`ch-content-share--${tileIdToRender} ${className || ''}`}
```

Comment:
is this a defined class name?

Reply:
No, this className is not used in fact, I added here to follow how we did for `RemoteVideo` component, it serves as an identifier
https://github.com/aws/amazon-chime-sdk-component-library-react/blob/efb7f0f4387a0929a1e2c9813a0afc57df1e2286/src/components/sdk/RemoteVideo/index.tsx#L50

Reply:
The prefix classname is to make sure we can confidently override the styles on this one and is a way to let builders know that they should not change these I believe. `ch` stands for `chime` :)